docs/guides/unity-catalog.md

```diff
@@ -191,17 +437,60 @@ resource "databricks_grants" "external_creds" {
 }
 
 resource "databricks_external_location" "some" {
+  provider        = databricks.workspace   
   name            = "external"
   url             = "s3://${aws_s3_bucket.external.id}/some"
   credential_name = databricks_storage_credential.external.id
   comment         = "Managed by TF"
 }
 
 resource "databricks_grants" "some" {
+  provider          = databricks.workspace   
   external_location = databricks_external_location.some.id
   grant {
     principal  = "Data Engineers"
     privileges = ["CREATE TABLE", "READ FILES"]
   }
 }
 ```
+
+## Configure Unity Catalog clusters
+
+To ensure the integrity of ACLs, users are required to access Unity Catalog through compute resources configured with strong isolation guarantees and other security features. This is achieved through a new concept called ‘Security Modes’, which are applied to compute resources.
+
+To use Unity Catalog from a regular cluster, one must configure the cluster with a proper **Data Security Mode**.  Currently, Unity Catalog is supported in two security modes: **User Isolation** and **Single User**.
+
+- **User Isolation** clusters can be shared by multiple users, but only SQL language is allowed. Some advanced cluster features such as library installation, init scripts and the DBFS Fuse mount are also disabled in this mode to ensure security isolation among cluster users.
+
+- To use those advanced cluster features or languages like Python, Scala and R with Unity Catalog, one must choose **Single User** Mode when launching the cluster. The cluster can only be used exclusively by a single user (by default the owner of the cluster); other users are not allowed to attach to the cluster.
+
+```hcl
+resource "databricks_cluster" "unity_sql" {
+  provider                = databricks.workspace
+  cluster_name            = "Unity SQL"
+  spark_version           = "10.1.x-scala2.12"
+  node_type_id            = "i3.xlarge"
```

Comment:
can you replace this one with `databricks_node_type` data resource?

Reply:
done